### PR TITLE
linux instructions for running two versions in parallel

### PR DESCRIPTION
**Pull request by wholenotes**
revised linux instructions for raspberry pi users who may be new to the build process and the terminal

### CODE REVIEW

File: docs/Build/Linux.md [hunk 1]
```diff
@@ -134,3 +134,36 @@ Copyright (c) 2008-2020 Juan-Pablo Caceres, Chris Chafe.
 SoundWIRE group at CCRMA, Stanford University
 ```
 
+### Running Two Versions of JackTrip in Parallel
+One level above the project directory of your current JackTrip installation, make another folder, enter it and clone the JackTrip repository again:
+
+```sh
+$ mkdir jacktrip-1.x.x
+$ cd jacktrip-1.x.x
+$ git clone --recurse-submodules https://github.com/jacktrip/jacktrip.git
```
Comment (ntonnaett) [inline docs/Build/Linux.md:143]: If you add jacktrip-1.x.x at the end here, you don't have to create the directory beforehand.

**Comment by dyfer** [comment]
I think the changes are okay (with the suggestion from @ntonnaett )
I just wanted to point out that this PR is pointing at the `main` branch. I personally think it's okay, so that these changes end up on the default branch right away, but after merging this PR we should also make a `main` -> `dev` merge soon so that possible subsequent changes to the `Linux.md` are preset on the `dev` branch.

**Comment by ntonnaett** [comment]
It also gets updated on push to `dev`. So let's use that as base.

**Comment by dyfer** [comment]
> It also gets updated on push to `dev`. So let's use that as base.

Ah, good point, I forgot this file is part of mkdocs. Let's change base to `dev` then.

**Comment by ntonnaett** [comment]
Do you want to update this,  @wholenotes?

File: docs/Build/Linux.md [hunk 2]
```diff
@@ -134,3 +134,37 @@ Copyright (c) 2008-2020 Juan-Pablo Caceres, Chris Chafe.
...
```
Comment (ntonnaett) [inline docs/Build/Linux.md:143]:
```suggestion
One level above the project directory of your current JackTrip installation, clone the JackTrip repository again in another directory (e.g. `jacktrip-1.x.x`):

```sh
$ git clone --recurse-submodules https://github.com/jacktrip/jacktrip.git jacktrip-1.x.x
```

File: docs/Build/Linux.md [hunk 3]
```diff
@@ -134,3 +134,37 @@ Copyright (c) 2008-2020 Juan-Pablo Caceres, Chris Chafe.
...
+```
+
+You now have two separate folders side by side: the new version in the jacktrip-1.x.x folder and the original one in the jacktrip folder.  You should be in the jacktrip-1.x.x folder now.  To build the new version without installing it:
+```sh
+$ cd jacktrip
```
Comment (ntonnaett) [inline docs/Build/Linux.md:148]:
```suggestion
You now have two separate folders side by side: the new version in the jacktrip-1.x.x folder and the original one in the jacktrip folder. To build the new version without installing it:
```sh
$ cd jacktrip-1.x.x
```

File: docs/Build/Linux.md [hunk 4]
```diff
@@ -134,3 +134,35 @@ Copyright (c) 2008-2020 Juan-Pablo Caceres, Chris Chafe.
...
+```
+
+You can switch back and forth between different JackTrip versions by running them in their respective build directories.
+
+Hint: If you lose track of where you are, this command will show your present working directory:
+```sh
+$ pwd
+```
+
+The new version's directory structure might look like this: ``` jacktrip-1.x.x/jacktrip/builddir``` and the old version ``` jacktrip/builddir```.
```
Comment (ntonnaett) [inline docs/Build/Linux.md:168]:
```suggestion
The new version's directory structure might look like this: ``` jacktrip-1.x.x/builddir``` and the old version ``` jacktrip/builddir```.
```

**Review by ntonnaett — APPROVED** [review]

**Pull request merged**